src/views/Samples/scss/index.scss

```diff
@@ -0,0 +1,115 @@
+@import "../../../globalStyles/scss/variables";
+
+.sample-page {
+  padding: 17px;
+  background-color: $alabaster;
+}
+
+.collection {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 34px;
+
+  &-wrapper {
+    display: flex;
+    align-items: center;
+
+    &__main {
+      display: flex;
+      align-items: center;
+
+      &-name {
+        color: $dull-orange;
+        font-size: 14px;
+      }
+
+      &-icon {
+        margin-right: 5px;
+
+        & > div {
+          display: flex;
+        }
+      }
+    }
+
+    &__grid {
+      margin-right: 32px;
```

Comment:
Use the variable $spacers or `rem` units for padding, margin.

Reply:
Done